awswrangler/cleanrooms/_read.py

```diff
@@ -23,10 +26,11 @@ def _delete_after_iterate(
 
 
 def read_sql_query(
-    sql: str,
-    membership_id: str,
-    output_bucket: str,
-    output_prefix: str,
+    sql: Optional[str] = None,
```

Comment:
Not a fan of having to set an empty as a default to required argument (e.g. membership_id). The only alternative is to break the order of arguments and push `sql` lower which I believe is worse